tests/vector_stores/test_postgres.py

```diff
@@ -167,7 +205,7 @@ async def test_add_to_db_query_and_delete(
         pg.add(node_embeddings)
     assert isinstance(pg, PGVectorStore)
 
-    q = VectorStoreQuery(query_embedding=[0] * 1536, similarity_top_k=1)
+    q = VectorStoreQuery(query_embedding=[0.1] * 1536, similarity_top_k=1)
```

Comment:
This test was periodically failing, and when I looked into it further, it looked like the cosine distance was being returned as NaN. From what I saw online, a zero vector cosine distance is undefined (which makes sense).